### PR TITLE
PR for working on API keys docs updates

### PR DESCRIPTION
**Pull request by danielone**
PR for working on API keys docs updates so Vectarans can work together on

### CODE REVIEW

**Comment by danielone** [comment]
@cjcenizal @eskibars any ideas why indexDocs is failing please for this PR?

File: www/docs/create_api_keys.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ Indexing and administrative actions cannot be performed through these keys.
 
 The remainder of this guide walks you through managing and using the API Keys.
 
+## Best Practices in Using API Keys
```
Comment (cjcenizal) [inline www/docs/create_api_keys.md:29]: Both this heading and the heading on line 31 are h2. To create a hierarchy, should this be an h1? Or should some of the headings that follow be changed to h3?

**Comment by cjcenizal** [comment]
I'm not sure what the `IndexDocs` task is supposed to do, but it completes with this message:

>   Error: Unable find a diff between main and 604dc3f1c1f5d436e3adaf700faa26dddf0c1411

I'm guessing this means that it's checking to see if a set of files got changed and, if not, then it's supposed to terminate because there's nothing for it to do. In this case I guess it's working as expected because the files that it's comparing don't have any changes.

**Comment by danielone** [comment]
@cjcenizal @eskibars @talip @aamirbutt Docs changes for api keys ready for review. 

@talip @aamirbutt - please review / add to PR for section Using an API Key b/c I have not updated anything there as more code / JWT related so you are experts there on needed changes.

File: www/docs/create_api_keys.md (hunk 1 above)
Comment (danielone) [reply www/docs/create_api_keys.md]: Headers adjusted in latest

Comment (danielone) [reply www/docs/create_api_keys.md]: Docsaurus uses following starting at H2 
https://docusaurus.io/docs/markdown-features/toc

## Level 2 title

### Level 3 title

#### Level 4 title

Level 1 for page title

**Comment by cjcenizal** [comment]
@danielone I think something went awry with this branch, because GitHub is showing over 5,000 files have been changed:
image

Typically I've seen this happen when a rebase or merge goes wrong. I suggest other reviewers hold off on further review while we sync up over Zoom and figure out a fix.

**Comment by cjcenizal** [comment]
This has been replaced by https://github.com/vectara/vectara-docs/pull/22

**Comment by aamirbutt** [comment]
It looks like CJ has already done a very thorough review. I will leave this to him and @talip .

**Pull request closed without merging**